test/unit/test.sign.js

```diff
@@ -173,6 +173,20 @@ describe("sign", function() {
     }).catch(done);
   });
 
+  it("can configure a proxy", function(done) {
+    runSignCmd({
```

Comment:
I know another test you probably modeled this on didn't do so, but it's easier to just return promises. Your test would change to the following. Notice how you have to drop the `done` argument in the callback for this to work.
````js
it("can configure a proxy", function() {
  return runSignCmd({
    ...
  }).then(function() {
    expect(signAddonCall.call[0].apiProxy)
      .to.be.equal("some-proxy");
  });
});
````